libraries/AP_Notify/OreoLED_PX4.h

```diff
@@ -89,6 +142,12 @@ class OreoLED_PX4 : public NotifyDevice
     oreo_state _state_desired[OREOLED_NUM_LEDS];    // desired state
     oreo_state _state_sent[OREOLED_NUM_LEDS];       // last state sent to led
     uint8_t _pattern_override;                      // holds last processed pattern override, 0 if we are not overriding a pattern
+    uint8_t _oled_theme;
+    uint8_t rear_color_r{255};                           // the rear LED red value
```

Comment:
our normal convention to initialise the variables is to use "=" unless it's an array.  So we would use, "rear_color_r = 255" instead of "rear_color_r{255}".
Also perhaps we should be consistent about using an unscore before the variable name?  We don't actually mind whether we use it or not, but let's be consistent within a class or file.  So perhaps, "rear_color_r" should be renamed to "_rear_color_r".